Vizzuality/heco-invest
# Create settings/info page and allow delete own user

This PR adds the page settings/information and the delete own account functionality 

## Description

The users should have a way to delete their own account.

There will be a new page ‘user-settings’

The page will display a button to delete the user account

A modal will be displayed to confirm the action

When the action is confirmed, the user will be logged-out and re directed to the home page


## Testing instructions

### Testing as a account user
1. Log in as a account user (not owner)
2. Go to settings. You will be redirected to settings/information
3. Click on 'delete' button on 'Delete my account' section. A confirmation modal should appear with a text explaining that you will lose your account and access to the platform.
4. Click on 'delete'. You should be signed out and redirected to the home page.

### Testing as account owner
1. Log in as a account owner
2. Go to settings. You will be redirected to settings/information
3. Click on 'delete' button on 'Delete my account' section. A confirmation modal should appear with a text explaining that you can't delete your account because you are the account owner.
5. The 'delete' button on the modal should be disabled.

## Tracking

[LET-808](https://vizzuality.atlassian.net/browse/LET-808)
